### PR TITLE
#12 POs should create issues

### PR DESCRIPTION
**Pull request by Cleop**
#12 Adds answer to why POs should create issues

### CODE REVIEW

**Review by nelsonic — APPROVED** [review]
👍

File: readme.md [hunk 1]
```diff
@@ -40,7 +40,18 @@ importance (although feel free to ask them to try to learn more about _why_
 12. Note, in order to be able to perform these responsibilities it's important
 that POs have been given the authority and means necessary to complete them. :muscle:
 
-* product backlog: a prioritised features list for the product found on github.
+*product backlog: a prioritised features list for the product found on github.
+
+## Product Owners should create issues rather than dwylers.
+
+### Why?
+
```
Comment (iteles) [inline readme.md]: The points in this section make sense but I'm feeling that it's a change in 'voice'. Whereas most of this repo is written as if speaking to POs, these bullet points feel like they're written _about_ POs but _not_ as if they were the ones reading this.

File: readme.md [hunk 2]
```diff
@@ -40,7 +40,18 @@ importance (although feel free to ask them to try to learn more about _why_
...
+- Fosters a clearer sense of ownership of the backlog
+- Allows us to pick up on nuances and possible communication issues early (e.g. expectations that weren't clearly communicated verbally, incorrect understandings of conversations had over other channels)
+- Engages POs with Github early in the process and ensures they know to go there for updates rather than emailing members of the team (for example)
+- They learn from being asked the same questions a few times by developers and improve their creation of issues over time
```
Comment (iteles) [inline readme.md]: This is a good example of what I mean, could come off as a little insulting...

**Review by iteles — CHANGES_REQUESTED** [review]
Just a thought, won't require big changes.

**Comment by Cleop** [comment]
@iteles - want to take a look at the new wording whenever you get a chance?

**Review by iteles — APPROVED** [review]
Thanks @Cleop 👍

**Pull request merged**